pkg/auth/model/model.go

```diff
@@ -109,16 +126,165 @@ func (s *Statements) Scan(src interface{}) error {
 	return json.Unmarshal(data, s)
 }
 
-type Credential struct {
-	AccessKeyID                   string    `db:"access_key_id"`
-	SecretAccessKey               string    `db:"-" json:"-"`
-	SecretAccessKeyEncryptedBytes []byte    `db:"secret_access_key" json:"-"`
-	IssuedDate                    time.Time `db:"issued_date"`
-	UserID                        int64     `db:"user_id"`
+const authPrefix = "auth"
```

Comment:
Document, prefix can mean a lot of things in general `model.go` file.

Reply:
done